pkg/api/handlers/feature_flags_handler.go

```diff
@@ -0,0 +1,70 @@
+package handlers
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	apierrors "github.com/Roll-Play/togglelabs/pkg/api/error"
+	"github.com/Roll-Play/togglelabs/pkg/models"
+	apiutils "github.com/Roll-Play/togglelabs/pkg/utils/api_utils"
+	"github.com/labstack/echo/v4"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+type FeatureFlagHandler struct {
+	db *mongo.Database
+}
+
+func NewFeatureFlagHandler(db *mongo.Database) *FeatureFlagHandler {
+	return &FeatureFlagHandler{
+		db: db,
+	}
+}
+
+func (ffh *FeatureFlagHandler) PostFeatureFlag(c echo.Context) error {
+	userID, err := apiutils.GetObjectIDFromContext(c)
+	if err != nil {
+		log.Println(apiutils.HandlerErrorLogMessage(err, c))
+		return apierrors.CustomError(
+			c,
+			http.StatusUnauthorized,
+			apierrors.UnauthorizedError,
+		)
+	}
+
+	orgID, err := primitive.ObjectIDFromHex(c.Param("orgID"))
+	if err != nil {
+		log.Println(apiutils.HandlerErrorLogMessage(err, c))
+		return apierrors.CustomError(
+			c,
+			http.StatusBadRequest,
+			apierrors.BadRequestError,
+		)
+	}
+
+	req := new(models.FeatureFlagRequest)
+	if err := c.Bind(req); err != nil {
+		log.Println(apiutils.HandlerErrorLogMessage(err, c))
+		return apierrors.CustomError(
+			c,
+			http.StatusBadRequest,
+			apierrors.BadRequestError,
+		)
+	}
+
+	model := models.NewFeatureFlagModel(ffh.db)
+	ffr, err := model.NewFeatureFlagRecord(req, orgID, userID)
+	newRecID, err := model.InsertOne(context.Background(), ffr)
+	if err != nil {
+		log.Println(apiutils.HandlerErrorLogMessage(err, c))
+		return apierrors.CustomError(c,
+			http.StatusInternalServerError,
+			apierrors.InternalServerError,
+		)
+	}
+	ffr.ID = newRecID
```

Comment:
this won't work. It will be defaulted to the zero value of objectID (000000000000...). Create a new objectID on the constructor function like in the other models

Reply:
Why do we even return it on our InsertOne functions?

Reply:
idk